### DISCUSSION TITLE
Kata 7 y error de Conexión en Dispositivo

### DISCUSSION THREAD

**Discussion by AleEysele24** [LaunchX-InnovaccionVirtual/CursoIntroPython#200] [General]
Estoy en la kata 7 y no entiendo que debo hacer porque  la info de la documentación es la misma que en la Kata 6.
Otra cosa, cuando quiero correr un programa me aparece que mi dispositivo no permitió la conexión, ¿Por qué pasa eso?.

**Comment by alfredo-inzunza** [comment]
Hola. El módulo 7 es referente a las estructuras de control `while` y `for` mientras que el módulo 6 es referente a operaciones con listas, tal vez renombraste mal algún archivo. En cuanto al error, sería útil si tomas una captura de pantalla y el contexto en que sucede.

Éxito.
